### PR TITLE
feat: putting theme in fontWeight bold and normal

### PR DESCRIPTION
**Pull request by CarolineLCa**
# Summary of Changes

### Expected behavior
Use any `fontFamily` you want in the application.

### Actual behavior
On some screens the `fontWeight` was directly on it, fonts that use numeric `fontWeight` were not working.

### To fix
I added the `fontWeight` directly from the `theme`

### Observation
I added `marginTop: 30` to the PINEnter screen image, as it was too high up, taking the risk of being cut off on some devices when the keyboard is shown.

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [X] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [X] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [X] Updated documentation as needed for changed code and new or modified features;
- [X] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: packages/legacy/core/App/screens/PINEnter.tsx [hunk 1]
```diff
@@ -302,6 +302,7 @@ const PINEnter: React.FC<PINEnterProps> = ({ setAuthenticated, usage = PINEntryU
               resizeMode: Assets.img.logoSecondary.resizeMode,
               alignSelf: 'center',
               marginBottom: 20,
+              marginTop: 30,
```
Comment (thiagoromanos) [inline packages/legacy/core/App/screens/PINEnter.tsx]: Could be this marging be themed too?

Comment (thiagoromanos) [reply packages/legacy/core/App/screens/PINEnter.tsx]: @CarolineLCa

**Review by thiagoromanos — CHANGES_REQUESTED** [review]
Check the comment about the PINEnter file

File: packages/legacy/core/App/screens/PINEnter.tsx [hunk 2]
```diff
@@ -70,6 +70,12 @@ const PINEnter: React.FC<PINEnterProps> = ({ setAuthenticated, usage = PINEntryU
       ...TextTheme.popupModalText,
       marginVertical: 5,
     },
+    image: {
+      ...PINEnterTheme.image,
```
Comment (CarolineLCa) [inline packages/legacy/core/App/screens/PINEnter.tsx:74]: @thiagoromanos  PINEnterTheme.image created

File: packages/legacy/core/App/screens/PINEnter.tsx (hunk 1 above)
Comment (CarolineLCa) [reply packages/legacy/core/App/screens/PINEnter.tsx]: > @CarolineLCa


> Could be this marging be themed too?

Done!

File: packages/legacy/core/App/screens/PINEnter.tsx (hunk 2 above)
Comment (thiagoromanos) [reply packages/legacy/core/App/screens/PINEnter.tsx:74]: Nice work. That can make the whole PINEnter screen more theme flexible.

**Review by thiagoromanos — APPROVED** [review]
LGTM

**Pull request merged**